developer-docs-site/docs/index.md

```diff
@@ -112,31 +112,38 @@ Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentatio
 </div>
 </div>
 
-## Aptos devnet
+## Connect to Aptos networks
+
+Aptos offers a local testnet and online devnet and testnet, with mainnet on the way. See the [System Integrators Guide](guides/system-integrators-guide.md#networks) for a summary of the available networks and the means to connect to them.
```

Comment:
Aptos offers running a local testnet, as well as using the shared devnet and testnet, with mainnet on the way.

Reply:
Done. Thanks!